lib/model/database.dart

```diff
@@ -52,30 +49,42 @@ class UriConverter extends TypeConverter<Uri, String> {
   @override Uri fromSql(String fromDb) => Uri.parse(fromDb);
 }
 
-LazyDatabase _openConnection() {
-  return LazyDatabase(() async {
-    // TODO decide if this path is the right one to use
-    final dbFolder = await getApplicationDocumentsDirectory();
-    final file = File(path.join(dbFolder.path, 'db.sqlite'));
-    return NativeDatabase.createInBackground(file);
-  });
-}
-
 @DriftDatabase(tables: [Accounts])
 class AppDatabase extends _$AppDatabase {
   AppDatabase(super.e);
 
-  AppDatabase.live() : this(_openConnection());
-
   // When updating the schema:
   //  * Make the change in the table classes, and bump schemaVersion.
-  //  * Export the new schema and generate test migrations:
+  //  * Export the new schema and generate test migrations with drift:
   //    $ tools/check --fix drift
+  //    and generate database code with build_runner.
+  //    See ../../README.md#generated-files for more
+  //    information on using the build_runner.
   //  * Write a migration in `onUpgrade` below.
   //  * Write tests.
   @override
   int get schemaVersion => 2; // See note.
 
+  Future<void> _resetDatabase(Migrator m) async {
+    // This should only ever happen in dev.  As a dev convenience,
+    // drop everything from the database and start over.
+    await m.database.transaction(() async {
+      final query = m.database.customSelect(
+        "SELECT name FROM sqlite_master WHERE type='table'");
+      for (final row in await query.get()) {
+        final data = row.data;
+        final tableName = data['name'] as String;
+        // Skip sqlite-internal tables, https://www.sqlite.org/fileformat2.html#intschema
+        // https://github.com/simolus3/drift/blob/0901c984a987e54ba0b67e227adbfdcf3c08eeb4/drift_dev/lib/src/services/schema/verifier_common.dart#L9-L22
```

Comment:
nit: I'm not sure how to interpret a comma here. How about:
```suggestion
        // Skip sqlite-internal tables.  See for comparison:
        //   https://www.sqlite.org/fileformat2.html#intschema
        //   https://github.com/simolus3/drift/blob/0901c984a/drift_dev/lib/src/services/schema/verifier_common.dart#L9-L22
```

Reply:
(That bit of Drift code is actually a bit puzzling:
```dart
/// Attempts to recognize whether [name] is likely the name of an internal
/// sqlite3 table (like `sqlite3_sequence`) that we should not consider when
/// comparing schemas.
bool isInternalElement(String name, List<String> virtualTables) {
  // Skip sqlite-internal tables, https://www.sqlite.org/fileformat2.html#intschema
  if (name.startsWith('sqlite_')) return true;
```

The example in the doc would *not* get recognized by the actual implementation. Unless it happens to be in the `virtualTables`, but I don't think that's what that argument is for.)

Reply:
Hmm, good catch. I believe that is a typo. The sqlite.org link does mention a table called `sqlite_sequence`, and nothing relevant comes up in my Google search for `"sqlite3_sequence"`.